packages/office-ui-fabric-react/src/utilities/positioning/positioning.ts

```diff
@@ -417,12 +417,13 @@ export namespace positioningFunctions {
     elementRectangle: Rectangle,
     hostElement: HTMLElement,
     targetEdge: RectangleEdge,
-    alignmentEdge?: RectangleEdge
+    alignmentEdge?: RectangleEdge,
+    coverTarget?: boolean
```

Comment:
Is it possible to test this, could you add an example with coverTarget: true to for e.g ContextualMenu examples

Reply:
I added a unit test that will catch this